src/components/widgets/PeopleField.vue

```diff
@@ -55,15 +62,22 @@ export default {
 
   created() {
     this.items = this.people
-    this.item = this.modelValue
     this.index = buildNameIndex(this.people)
   },
 
   mounted() {
     this.items = this.people
+    this.item = this.modelValue
+    setTimeout(() => {
+      this.item = this.modelValue
+    }, 10)
```

Comment:
Why this code?

Reply:
Because the item is not properly selected on mount. The value is right but the person name doesn't appear in the widget.